.renovaterc.json

```diff
@@ -83,6 +83,16 @@
       }
     },
 
+    {
+      "packagePatterns": ["\\bbabel"],
```

Comment:
Curious why the `\b` part is needed here?

Reply:
It enforces word boundaries, which is probably worth sticking to. (I used the precedent set by the pattern for `prettier|eslint`.)

Reply:
SGTM